### ISSUE TITLE
Add an option to export classes as named exports instead of default export

### ISSUE THREAD

**Issue by finom** [webpack-contrib/css-loader#469]
**Do you want to request a *feature* or report a *bug*?**

*feature*

**What is the current behavior?**

Styles imported as default import object.

```js
import css from './style.css';
```

**What is the expected behavior?**

Styles imported as named imports.

```js
import * as css from './style.css'
```

**If this is a feature request, what is motivation or use case for changing the behavior?**

Webpack 2 warns user when wrong reference is imported/used.

```js
import * as all  from './all';

console.log(all.notExportedReference); // webpack 2 warns
```

I'd like to see the same behavior when CSS modules are used to get rid of all undefined class names.

```js
<div className={css.notExportedClassName} />
```

**Comment by michael-ciniawsky** [comment]
@finom Currently `css-loader` exports CSS as CJS Modul, this will be addressed in `v1.0.0` when we export CSS as ES2015 Modules, but it's a breaking change alongside with other refactors in the pipeline, so please be patient 😛

**Comment by evilebottnawi** [comment]
Look comment above :+1:
